SurveyTests/Domain/Use Case/Login/StoreTokenUseCaseSpec.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  StoreTokenUseCaseSpec.swift
+//  SurveyTests
+//
+//  Created by Khanh on 09/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Nimble
+import Quick
+@testable import Survey
+
+final class StoreTokenUseCaseSpec: QuickSpec {
+
+    override func spec() {
+
+        var storeTokenUseCase: StoreTokenUseCase!
+        var keychain: KeychainService!
+
+        describe("a StoreTokenUseCase") {
+
+            beforeEach {
+                keychain = KeychainService(service: "")
```

Comment:
Please create a service with a name for Unit test. But if you're planning to use protocol DI then you might not need service naming at all 😄